### PR TITLE
CHE-2594: Adding 'build-simple' profile for building Che faster by skipping unit tests, license checks and other enforcement features

### PR DESCRIPTION
**Pull request by ibuziuk**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2078,5 +2078,17 @@
                 </plugins>
             </build>
         </profile>
+        <!-- Profile for building Che faster by skipping unit tests, license checks and other enforcement features -->
+        <profile>
+            <id>build-simple</id>
```
Comment (benoitf) [inline pom.xml]: the name is not obvious for me

**Comment by ibuziuk** [comment]
@benoitf @TylerJewell what about changing the name to "fast-and-dirty" and moving profile to https://github.com/ibuziuk/che ?

**Comment by benoitf** [comment]
name is more obvious this time :-)

> moving profile to https://github.com/ibuziuk/che ?
I think you're referencing eclipse/che (not your fork)

for me it may be better place but I listen to others.

**Comment by ibuziuk** [comment]
> I think you're referencing eclipse/che (not your fork)

@benoitf sure :-)

**Comment by TylerJewell** [comment]
How about we just name the profile "fast".  The "dirty" part makes me think of naughty thoughts from high school and college days.

**Comment by ibuziuk** [comment]
@TylerJewell I'm ok with "fast" :-)

**Comment by benoitf** [comment]
I'm ok with both names as long we don't get into "fast and furious"

**Comment by ibuziuk** [comment]
closing assuming https://github.com/eclipse/che/pull/2733 has been merged

**Pull request closed without merging**